src/lib/api/axiosInstance.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file axiosInstance.js
+ * Exports a method that assists in communicating with EditVR's API.
+ */
+
+import axios from 'axios';
+
+import { apiURL } from '../../config';
+import {
+  ERROR_API_CONNECTION_FAILED,
+  ERROR_API_FORBIDDEN,
+  ERROR_API_GENERAL,
+  ERROR_API_NOT_FOUND,
+  ERROR_API_LOGIN_FAILED,
+  ERROR_API_BAD_REQUEST,
+  AXIOS_ERROR_NETWORK,
+  AXIOS_ERROR_400,
+  AXIOS_ERROR_401,
+  AXIOS_ERROR_403,
+  AXIOS_ERROR_404
+} from '../../constants';
+
+/**
+ * Creates an Axios instance configured to connect to the EditVR API.
+ *
+ * @param {object} authentication - Auth config for this method.
+ * @param {string} authentication.accessToken - Access token for this request.
+ * @param {string} authentication.csrfToken - CSRF-Token token for this request.
+ * @param {string} format - API format. Default is jsonapi.
+ *
+ * @returns {object} - Axios instance.
+ */
+const axiosInstance = (
```

Comment:
This generates an axiosInstance that currently handles adding auth info, content type headers, etc to requests. It also adds interceptors that turn Axios errors into nice EditVR errors that make more sense.